snippets/auth_ecr.erb

```diff
@@ -0,0 +1,11 @@
+auth_ecr() {
+  $(aws ecr get-login --region us-east-1 --no-include-email)
+}
+
+worked=1
+for attempt in {1..200}; do
+  [[ $worked != 0 ]] || break
+  auth_ecr && worked=0 || (log "ecr auth: attempt $attempt failed, sleeping 30"; sleep 30)
```

Comment:
This will try to auth for what? 100 minutes? Seems like a long time. I'm curious why it's so long! Is ECR really unreliable?

Reply:
Honestly, was working off of the timing from the existing snippets! ECR should be pretty reliable, so I'm ok with bringing this down by a lot.

Reply:
Yea, we can definitely bring it down. I'm not sure why we had it so high. I think the deploy times out well before this would anyway, so it was mostly just to handle scenarios where the master build failed strangely or something.